cypress/e2e/clue/branch/student_tests/expression_tool_spec.js

```diff
@@ -4,6 +4,7 @@ import ExpressionToolTile from '../../../../support/elements/clue/ExpressionTool
 let clueCanvas = new ClueCanvas;
 let exp = new ExpressionToolTile;
 
+// FIXME: skipping tests for now to get this deployed
```

Comment:
You should remove this line too.

Reply:
Looks like this is already removed, thanks.